docs/stories/getting-started/usage-examples/top-app-bar-as-container.stories.js

```diff
@@ -0,0 +1,104 @@
+import '@vonage/vwc-side-drawer';
+import '@vonage/vwc-top-app-bar';
+import '@vonage/vwc-top-app-bar-fixed';
+import '@vonage/vwc-icon';
+import '@vonage/vwc-layout';
+import '@vonage/vwc-list';
+import '@vonage/vwc-list/vwc-list-item';
+import '@vonage/vwc-list/vwc-list-expansion-panel';
+import { html } from 'lit-element';
+import { spread } from '@open-wc/lit-helpers';
+
+export default {
+	title: 'Getting Started/Usage Examples/Top App Bar',
+};
+
+const style = html`
+	<style>
+		.sb-show-main.sb-main-padded {
+			padding: 0;
+		}
+		p {
+			color: "#C0C0C0";
```

Comment:
the style doesn't work because of the "". Anyway the text is too light to be on white surface. 
Again - here toy can wrap the text "SECTION TITLE" with `<vwc-text font-face="body-1-bold">` with P inside or outside :) and then you will not need the P style here